docs/features/execution_payload_retention.md

```diff
@@ -10,42 +10,54 @@ This allows access via the API (or optionally direct DB/Elasticsearch querying)
 
 ## Payload record cleanup
 
-To reduce storage requirements, a CloudWatch rule (`{stack-name}-dailyExecutionPayloadCleanupRule`) triggering a daily run of the provided cleanExecutions lambda has been added.  This lambda will remove all 'completed' and 'non-completed' payload records in the database that are older than the specified configuration.
+To reduce storage requirements, a CloudWatch rule (`{stack-name}-dailyExecutionPayloadCleanupRule`) triggering a daily run of the provided cleanExecutions lambda has been added.  This lambda will remove a batch of payload records in the database that are older than the specified configuration.
+
+### Asynchronous es task
+
+The cleanExecutions lambda launches an asynchronous elasticsearch cleanup task which can be accessed from outside of the lambda function.
```

Comment:
`task` 

Nit: is 'task' the right phrase?    Is this a 'query' or a 'request'?      We should make sure not to accidentally conflate this with other things.

Reply:
Talked with Ethan, this is consistent with ES docs.